src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationServiceTest.java

```diff
@@ -958,6 +991,30 @@ public void throwExceptionWhenContestedGeneralApplicationReferToJudgeEmailIsRequ
         }
     }
 
+    @Test
+    public void sendContestedGeneralApplicationOutcomeNotificationEmailWhenSendToFRCToggleTrue() throws IOException {
+        when(featureToggleService.isSendToFRCEnabled()).thenReturn(true);
+        callbackRequest = getContestedCallbackRequest(MIDLANDS, MIDLANDS_FRC_LIST,
+            NOTTINGHAM, NOTTINGHAM_COURTLIST,  "FR_s_NottinghamList_1");
+
+        mockServer.expect(MockRestRequestMatchers.requestTo(END_POINT_CONTESTED_GENERAL_APPLICATION_OUTCOME))
+            .andExpect(MockRestRequestMatchers.method(HttpMethod.POST))
```

Comment:
can you update these 2 days to expect the correct value for notification email as done here https://github.com/hmcts/finrem-case-orchestration-service/blob/426ee347632148725460b1957cb373a0a0dccf8b/src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationServiceTest.java#L969